waiter/test/waiter/scheduler/marathon_test.clj

```diff
@@ -583,7 +583,7 @@
         service-id "service-1"
         instance-id "service-1.A"
         make-marathon-scheduler #(->MarathonScheduler {} {} (fn [] nil) "/home/path/"
-                                                      (atom {}) %1 %2 (constantly true))
+                                                      (atom {}) %1 {} %2 (constantly true))
```

Comment:
Can we please extract the map in a common variable and name it `service-id->service-description-fn`? Seeing the map is confusing when I was expecting to have a function there. We should be able to use the common variables in all the map occurrence below.

Reply:
Switched to `(constantly nil)` as per our discussion.